specification/components/schemas/responses/changeShortlistAndSendForTriage/422Response.yaml

```diff
@@ -0,0 +1,22 @@
+description: |
+  Where status code 422 (Unprocessable Entity) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below.
+  Check diagnostic property for specific information regarding the error.
+
+  | Error code                         | Description                                                                                                             |
+  | ---------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+  | INAPPROPRIATE_VALUE                | A value, which is acceptable under different conditions, is inappropriate in the context of the other information provided. |
+  | TOO_MANY_ITEMS                     | In a list where a maximum number of items is specified (e.g. a Shortlist), too many entries are supplied. |
+  | MISSING_VALUE                      | A field defined as mandatory for an endpoint has not been provided. |
+  | PATIENT_ERROR                      | An error occurred while retrieving the requested patient. Do not attempt again. |
+  | INVALID_VALUE                      | The input provided does not conform to the expected data types and format. |
+  | INVALID_CODE                       | The input provided for a field is not one of the defined legal values. |
+  | UNEXPECTED_FIELD                   | A field is provided that is not expected as part of the request. E.g. a field is mis-spelt, was defined on a previous version of the endpoint but has subsequently been removed. |
+  | INVALID_FHIR_STRUCTURE             | The FHIR data structure in the message body does not match the expected structure (e.g. an array is present for a value when no array is expected). |
+  | ORGANISATION_IS_CLOSED             | The organisation supplied corresponds to an organisation that is closed. |
+  | ORGANISATION_NOT_APPROPRIATE       | The organisation supplied corresponds to an organisation that is not valid for the given request. |
+  | DUPLICATE_SERVICE                  | Unique service value expected. |
```

Comment:
As the shortlist is only allowed to include a single service, I don't think this outcome will be returned.